<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlidePushButton</class>
 <widget class="QWidget" name="SlidePushButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>176</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>188</width>
    <height>176</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>188</width>
    <height>176</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#label_name{
	font-size:20px;
	color:green ;
	font-weight:bold ; 
}
QLabel#label_introduce{
	font-size: 12px ; 
	color:rgb(112,112,112) ; 
}

QLabel#label_image{
		border-style: groove;
		border-width: 4px ; 
		border-color: 	rgb(128,128,128) ;
}


QFrame[frameShape=&quot;4&quot;]#line_split{
	border-width: 2px; 
	border-top-style: none;
	border-right-style: none; 
	border-left-style: none;
	border-bottom-style: solid; 
	border-color:rgb(184,184,184);
}

QFrame[frameShape=&quot;4&quot;]#line_name{
	border-width: 2px; 
	border-top-style: none;
	border-right-style: none; 
	border-left-style: none;
	border-bottom-style: solid; 
	border-color:rgb(184,184,184);
}

QWidget#SlidePushButton{
	border-width:3px ; 
	border-style:solid ;
	border-radius:10px ;
	border-color:rgb(120,120,120) ;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_main">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_top">
       <item>
        <widget class="QLabel" name="label_image">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>81</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_split">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>161</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="midLineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_introduce">
         <property name="minimumSize">
          <size>
           <width>156</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_bottom">
       <item>
        <widget class="Line" name="line_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>163</width>
           <height>0</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>163</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
